openstates/az/bills.py

```diff
@@ -213,6 +215,51 @@ def actor_from_action(self, bill, action, self_chamber):
                 else:
                     return action_map[key]
 
+    def scrape_votes(self, bill, page):
+        base_url = 'https://apps.azleg.gov/api/BillStatusFloorAction'
+        for header in page['FloorHeaders']:
+            params = {
+                'billStatusId': page['BillId'],
+                'billStatusActionId': header['BillStatusActionId'],
+                'includeVotes': 'true',
+            }
+            resp = self.get('{}?{}'.format(base_url, urllib.parse.urlencode(params)))
```

Comment:
Under the hood, `self.get` is from the [requests library](http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls). So you can just do this as

```python
resp = self.get(base_url, params=params)
```